### PR TITLE
refactor: crda to rhda alpha

### PR DESCRIPTION
**Pull request by IlonaShishov**

### CODE REVIEW

File: .github/workflows/stage.yml [hunk 1]
```diff
@@ -5,30 +5,42 @@ on:
   workflow_dispatch:
   push:
     branches:
-      - main
+      - master
     paths-ignore:
```
Comment (zvigrinberg) [inline .github/workflows/stage.yml:9]: @IlonaShishov you might consider trigger a workflow also if you're changing the workflow itself...

File: .github/workflows/stage.yml [hunk 2]
```diff
@@ -5,30 +5,42 @@ on:
...
+          echo "@fabric8-analytics:registry=https://npm.pkg.github.com" > ~/.npmrc
+      
       - name: Install @vscode/vsce
         run: npm i -g @vscode/vsce
 
       - name: Install Dependencies
         run: npm ci
 
-      - name: Compile
+      - name: Compile for test
```
Comment (zvigrinberg) [inline .github/workflows/stage.yml:43]: @IlonaShishov Is this working?? i  remember that we had problems with compildation for tests didn't we?

**Review by zvigrinberg — COMMENTED** [review]
Hi @IlonaShishov 
Could you just refer to my comments before i'm approving?

Thanks.

File: .github/workflows/stage.yml (hunk 1 above)
Comment (IlonaShishov) [reply .github/workflows/stage.yml:9]: you are correct, will make modifications, thanks

**Review by zvigrinberg — APPROVED** [review]

File: .github/workflows/stage.yml (hunk 2 above)
Comment (IlonaShishov) [reply .github/workflows/stage.yml:41]: yes, 
we are using a workaround for the incompatibility between CommonJS and ESM modules and temporarily not testing sections of code invoking the exhort-javascript-api dependency.

**Pull request merged**